R/deprec-do.r

```diff
@@ -47,7 +49,7 @@
 #' }
 do <- function(.data, ...) {
   lifecycle::deprecate_warn("1.0.0", "do()",
-    details = "Use condense() or summarise()"
+    details = "See ?do for translation help"
```

Comment:
```suggestion
    details = "See ?do for advice on how to update"
```

Just a suggestion. Does "translation" seem like the wrong word here?